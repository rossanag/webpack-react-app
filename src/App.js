import '../styles/styles.css';

const App = () => {
    return (
      <div className='container'>
        <h1>Hello!!</h1>
        <h2>Welcome to your First React App..!</h2>
        <h4>(created wihthout create-react-app)</h4>
      </div>
    )  
}

export default App;
