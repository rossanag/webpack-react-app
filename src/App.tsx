// import React from 'react';

const App = () => {
    return (
      <div className='container'>        
        <h1 className="text-4xl text-amber-300">Hello!!</h1>
        <h2 className="text-2xl text-rose-600">Welcome to your First React App..!</h2>           
        <h5>(created wihthout create-react-app)</h5>
      </div>
    )  
}

export default App;
