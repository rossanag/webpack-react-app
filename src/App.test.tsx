import { render, screen } from '@testing-library/react';
import App from './App';

/* describe('Testing example', () => {
  it('renders Title Hello', () => {
    render(  <App/>);
    const textElement = screen.getByText(/Hello!!/i);
    expect(textElement).toBeInTheDocument();
  });
}) */

describe('Testing example', () => {
  it('renders Title Hello', () => {
    const {container } = render(  <App/>);
    // const textElement = .getByText(/Hello!!/i);
    // expect(textElement).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });  
})

