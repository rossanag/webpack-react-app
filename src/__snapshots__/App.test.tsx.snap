// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing example renders Title Hello 1`] = `
<div>
  <div
    class="container"
  >
    <h1
      class="text-4xl text-amber-300"
    >
      Hello!!
    </h1>
    <h2
      class="text-2xl text-rose-600"
    >
      Welcome to your First React App..!
    </h2>
    <h5>
      (created wihthout create-react-app)
    </h5>
  </div>
</div>
`;
